docs/api/structures/cpu-usage.md

```diff
@@ -2,6 +2,7 @@
 
 * `percentCPUUsage` number - Percentage of CPU used since the last call to getCPUUsage.
   First call returns 0.
+* `cumulativeCPUUsage` number - Total seconds of CPU time used since process startup.
```

Comment:
should be marked as `optional` since its presence is conditional on the return value of `GetCumulativeCPUUsage()`

```suggestion
* `cumulativeCPUUsage` number (optional) - Total seconds of CPU time used since process startup.
```

Reply:
Done.